README.md

```diff
@@ -62,7 +62,18 @@ Spotlight indexing on OS X can result in multiple events (see [howeyc #62][#62])
 **How many files can be watched at once?**
 
 There are OS-specific limits as to how many watches can be created:
-* Linux: /proc/sys/fs/inotify/max_user_watches contains the limit, reaching this limit results in a "no space left on device" error.
+* Linux: The following files have an influence on the overall limit:
```

Comment:
This limit sounds like it's for the number of watchers and not the number of files? If so, I think it should be a separate question about the number of watchers.

Reply:
Its generally only about the number of watchers, so I guess we're fine here right? :)